powerspy.py

```diff
@@ -127,6 +131,8 @@ def recvCmd(self, size = 1):
         buf = mat.group(1)
         logging.debug("RECV: <%s>" % buf)
         break
+    buf = buf.replace("'b'", "") # Fix receiving 'b' on some powerspy connections
```

Comment:
That's strange. I don't have access to PowerSpy device so I will trust you on this.

Reply:
Most of the time, the received buffer from recvCmd gives me 'b' interlaced in between the data. For example: ```DEBUG:root:RECV: <P'b'O'b'W'b'E'b'R'b'S'b'P'b'Y'b'C'b'0'b'1'b'0'b'0'b'4'b'0'b'1'b'0'b'0'b'0'b'0'b'1'b'>```
Hence, why I just added the simple replace.

Reply:
That's weird. Are you sure this is not due to python3 here?

Reply:
I'm having it in Python 2 also on Linux.